rdt/transformers/null.py

```diff
@@ -81,22 +82,21 @@ def transform(self, data):
         Returns:
             numpy.ndarray
         """
-        if self.nulls:
```

Comment:
Does this have anything to do with the performance test failures?

Reply:
Hmm, maybe, let me try.

Reply:
We can't roll back the change that easily, because the previous code was faulty. I added a check before transforming the data to series, which made the code slightly faster, but not significantly. I'll try to see if I can speed it up somewhere else.